app/config/sculpin_site.yml

```diff
@@ -46,6 +46,9 @@ subspecies:
     magento:
         common: "Magento"
         latin: "mercatoris"
+    office-partner:
+        common: "Office Partner"
+        latin: "officium socium"
```

Comment:
The subspecies should be written as a single word--"officiumsocium" would be fine.